oedi/config.yaml

```diff
@@ -50,6 +94,7 @@ AWS:
         - s3://oedi-data-lake/pvdaq/parquet/mount/
         - s3://oedi-data-lake/pvdaq/parquet/other-instruments/
         - s3://oedi-data-lake/pvdaq/parquet/pvdata/
+      Table Prefix: pv_daq_
```

Comment:
Change to `pvdaq_`, it's the project name used all time.

Reply:
fixed